docs/user/openstack/customization.md

```diff
@@ -90,6 +91,37 @@ pullSecret: '{"auths": ...}'
 sshKey: ssh-ed25519 AAAA...
 ```
 
+## Image Overrides
+
+When doing a disconnected installation, the OpenStack platform has the additional requirement that we have locations to download the RHCOS images. The installer downloads these from a location described in
+[/data/data/rhcos.json](/data/data/rhcos.json), but they can be overridden to point to a local mirror.
+
+To do so and upload binary data from a custom location the user may set `clusterOSImage` parameter in the install config that points to that location, and then start the installation. In all other respects the process will be consistent with the default.
+
+**NOTE:** For this to work, the parameter value must be a valid http(s) URL.
+
+**NOTE:** The optional `sha256` query parameter can be attach to the URL, which will force the installer to check the image file checksum before uploading it into Glance.
+
+Example:
+
+```yaml
+platform:
+  openstack:
+      clusterOSImage: http://mirror.example.com/images/rhcos-43.81.201912131630.0-openstack.x86_64.qcow2.gz?sha256=ffebbd68e8a1f2a245ca19522c16c86f67f9ac8e4e0c1f0a812b068b16f7265d
+```
+
+If the user wants to reuse an existing Glance image without any uploading of binary data, then it is possible to set `clusterOSImage` install config parameter that specifies the Glance image name. In this case no new Glance images will be created, and the image will stay when the cluster is destroyed.
+
+Example:
+
+```yaml
+platform:
+  openstack:
+      clusterOSImage: my-rhcos
+```
+
+**NOTE:** The only difference in behavior with the previous example is that the value here is not an http(s) URL.
```

Comment:
It's not the only difference! The sha256 checksum is missing as well. I'd say instead:
```
If `clusterOSImage` is not an URL, then the installer will look into Glance for an image with that name.
```

Reply:
done